third_party/move/move-binary-format/src/file_format.rs

```diff
@@ -1162,9 +1162,6 @@ impl SignatureToken {
             (SignatureToken::Reference(ty1), SignatureToken::Reference(ty2)) => {
                 ty1.is_assignable_from(ty2)
             },
-            (SignatureToken::MutableReference(ty1), SignatureToken::MutableReference(ty2)) => {
```

Comment:
nit: i think the documentation for this method is out of date, it only specifies function types